lib/plox/date_scale.ex

```diff
@@ -1,18 +1,46 @@
 defmodule Plox.DateScale do
   @moduledoc """
-  A scale for elixir `Date` values
+  A scale of date values (`t:Date.t/0`).
 
   This struct implements the `Plox.Scale` protocol.
+
+  `Plox.Scale.values/2` returns a `t:Date.Range.t/0` enumerable:
+
+    iex> scale = Plox.DateScale.new(Date.range(~D[2020-01-01], ~D[2020-01-10], 1))
+    iex> scale |> Plox.Scale.values(%{step: 2}) |> Enum.to_list()
+    [~D[2020-01-01], ~D[2020-01-03], ~D[2020-01-05], ~D[2020-01-07], ~D[2020-01-09]]
+
+    iex> scale = Plox.DateScale.new(Date.range(~D[2020-01-10], ~D[2020-01-01], -1))
+    iex> scale |> Plox.Scale.values(%{step: 3}) |> Enum.to_list()
+    [~D[2020-01-10], ~D[2020-01-07], ~D[2020-01-04], ~D[2020-01-01]]
+
+  `Plox.Scale.convert_to_range/3` returns a number in the given range:
+
+    iex> scale = Plox.DateScale.new(Date.range(~D[2020-01-01], ~D[2020-01-09], 1))
+    iex> Plox.Scale.convert_to_range(scale, ~D[2020-01-05], 0..100)
+    50.0
+
+    iex> scale = Plox.DateScale.new(Date.range(~D[2020-01-09], ~D[2020-01-01], -1))
+    iex> Plox.Scale.convert_to_range(scale, ~D[2020-01-07], 0..100)
+    25.0
   """
   defstruct [:range]
 
   @type t :: %__MODULE__{}
 
   @doc """
-  Creates a new `Plox.DateScale` struct
+  Creates a new `Plox.DateScale` struct.
+
+  Raises if `range` is not a `t:Date.Range.t/0` struct or if it does not contain at least
+  two dates. The step is ignored. Supports forward and backward ranges.
```

Comment:
```
  two dates. The step is ignored. Supports forward and backward ranges.
```

Which "step" is being referred to here? I think it's referring to the `step` parameter that can be passed to `Range.new/3` -- can you please confirm?